### ISSUE TITLE
Gradle plugin has no default propertyWordDelimiters

### ISSUE THREAD

**Issue by asker123** [joelittlejohn/jsonschema2pojo#162]
Hi All,

I have been using this library for converting my jsonschema2pojo and I use both maven and gradle for my project. 

My JsonSchema looks like below:
```json
{
	"javaType" : "user.Tool",
	"type":"object",
	"$schema": "http://json-schema.org/draft-03/schema",
	"id": "http://jsonschema.net",
	"required":false,
	"properties":{
		"tool_key": {
			"type":"number",
			"id": "http://jsonschema.net/tool_key",
			"required":false
		},
		"tool_password": {
			"type":"string",
			"id": "http://jsonschema.net/tool_password",
			"required":false
		}
	}
}
```
For maven, it generates java class like: ToolKey and ToolPassword
For gradle, it generates java class like: Tool_key and Tool_password

Is this a problem with using this library? Is there any solution to have the pojo's created with same names?

Please let me know. Appreciate any help.

Thanks.

**Comment by ben-manes** [comment]
Probably the Gradle plugin's `JsonSchemaExtension` should initialize the values by creating a `DefaultGenerationConfig` and querying it for each field. The fields are currently set to manually after inspecting `DefaultGenerationConfig`. As the default config was changed, the Gradle plugin's defaults were out of sync. Using `DefaultGenerationConfig` directly would ensure that it is the canonical truth.

**Comment by joelittlejohn** [comment]
Very good point. This is similar to what the website does (
jsonschema2pojo.org) to maintain defaults for anything that's not
explicitly specified.

**Comment by joelittlejohn** [comment]
Fixed by #625.
